### ISSUE TITLE
Error in logic between DCs and Save Attempts that allows abuse of the system

### ISSUE THREAD

**Issue by I-mess-up** [tagpro-game/tagpro-issues#253]
Right now, players can have a free disconnect every 8 hours in the first 30 seconds of a game. The free DC does not negatively impact one's win percentage.

Save attempts also exist, which arise by chance but, once in the game, are a no-lose situation.

The problem is that, by combining these two, it is possible to take advantage and create long-term no lose situations. Basically, a player can play a game with stats off, then while waiting for the next game to start, enable stats. This creates the possibility for a save attempt. Then if the player does not have a save attempt in the next game, they immediately disconnect and wait 8 hours to try again. If and when the player successfully joins a save attempt game, they either win or lose, but it doesn't affect the win percentage.

TL;DR: Try to get a save attempt. If no, disconnect for free. If yes, play. 100% wins.

**Comment by CocoaBotter** [comment]
I think that this is a feature, not a bug.
In order to get a perfect rolling 300, it would take you well over 3 months, assuming that there are 50/50 odds of getting a save attempt. (given the realistic odds are much lower, you might as well learn to play tagpro well, for much less effort.

**Comment by I-mess-up** [comment]
Regardless of how long it would take, it's cheating. I'd like to hear a stance from the devs on this.

**Comment by Nick-Riggs** [comment]
We realized it would be possible when we implemented these features and we talked about it. We landed on even if someone was patient enough to do it, we (or players) would likely notice someone with a super high win percentage that barely played. And we could delete the account.
